package/tests/conftest.py

```diff
@@ -16,6 +16,11 @@
 from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
 from kedro_viz.server import populate_data
 
+try:
+    from kedro.io import MemoryDataset
```

Comment:
```suggestion
    # kedro 0.18.11 onwards
    from kedro.io import MemoryDataset
```

Reply:
Are these changes in tests really necessary? @astrojuanlu if you have a view; I don't think I saw you change in tests over in `kedro-plugins`